format_spec/fragment.md

```diff
@@ -186,6 +186,17 @@ The fragment min max sum null count is a [generic tile](./generic_tile.md) with
 | Sum | `uint64_t` | Sum value for attribute/dimension N |
 | Null count | `uint64_t` | Null count value for attribute/dimension N |
 
+The processed conditions is a [generic tile](./generic_tile.md) with the following internal format:
+
+| **Field** | **Type** | **Description** |
+| :--- | :--- | :--- |
+| Num | `uint64_t` | Number of processed conditions |
+| Consition size | `uint64_t` | Condition size 1 |
+| Condition | `char` | Condition value 1 |
```

Comment:
Documentation point: IIUC this is the _relative URI_ of the serialized QC file representing the operation(s) which have been applied to this fragment. If that is correct, suggest `Condition value 1` -> `Condition marker filename 1` etc.